website/docs/automation/controlplanes/ack/provision-resources.md

```diff
@@ -1,70 +1,93 @@
+![Trusted Entities](./assets/trust-entities.png)
+
+---
+
+
+Now, let's explore how we'll create the DynamoDB Table via a Kubernetes manifest
 
 ```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-security-group.yaml
+manifests/modules/automation/controlplanes/ack/dynamodb/dynamodb-create.yaml
 ```
 
 :::info
 
-The EC2 security group above allows any traffic from the CIDR range of the VPC used by the EKS cluster. This has been done to keep the example clear and understandable. A more secure approach would be to use [Security Groups for Pods](../../../networking/security-groups-for-pods/index.md) to allow traffic from specific pods.
+Astute readers will notice the YAML Spec to be similar to the API endpoints and calls for DynamoDB such as ```tableName``` and ```attributeDefinitions```.
 
 :::
 
-Next we want the RDS database to use the private subnets in our VPC. To accomplish this, we'll create a `DBSubnetGroup` which selects the appropriate subnet IDs:
+Next, we will need to update the regional endpoint for DynamoDB within the Configmap used by the Kustomization to update the **Carts** deployment.
 
 ```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-dbgroup.yaml
+manifests/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml
 ```
 
-Finally, we can create the configuration for the RDS database itself with a `DBInstance` resource:
+Next we want to populate the environment variable within the manifest. Run
 
-```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-instance.yaml
-```
+```bash
 
-Apply this configuration to the Amazon EKS cluster:
+$ envsubst < ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml \
+  | tee ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml \
+  > /dev/null
 
-```bash wait=30
-$ kubectl apply -k ~/environment/eks-workshop/modules/automation/controlplanes/ack/rds/k8s
-securitygroup.ec2.services.k8s.aws/rds-eks-workshop created
-dbinstance.rds.services.k8s.aws/rds-eks-workshop created
-dbsubnetgroup.rds.services.k8s.aws/rds-eks-workshop created
 ```
 
-The ACK controllers in the cluster will react to these new resources and provision the AWS infrastructure it has expressed. For example, we can use the AWS CLI to query the RDS database:
+:::info
+This process uses the ```envsubst``` utility to rewrite the environment variable into the manifest to reflect the 
+current region.
+:::
+
 
-```bash
-$ aws rds describe-db-instances \
-    --db-instance-identifier ${EKS_CLUSTER_NAME}-catalog-ack
+We're now finally able to apply these configurations to the Amazon EKS cluster:
+
+```bash wait=30
+$ kubectl apply -k ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb
+configmap/carts-ack created
+table.dynamodb.services.k8s.aws/items created
+deployment.apps/carts configured
```

Comment:
When you update a deployment please use `kubectl rollout status` to make sure it blocks until the update is completed.

Reply:
Done!